yetanotherco/aligned_layer
# fix (aggregator): wait one block before submitting response onchain

Aggregator waits for a block before submitting response

# To test

Locally, test normal flow, you can increase block time / run anvil without block time and send txs manually to get new block to check aggregator does not send response until a new block is reached.

Code is running on stage. http://stage.alignedlayer.com
